qa/standalone/special/ceph_objectstore_tool.py

```diff
@@ -453,7 +453,7 @@ def set_osd_weight(CFSD_PREFIX, osd_ids, osd_path, weight):
     assert(ret == 0)
 
     # osdmaptool increases the epoch of the changed osdmap, so we need to force the tool
-    # to use use a different epoch than the one in osdmap
+    # to use a different epoch than the one in osdmap
     cmd = CFSD_PREFIX + "--op set-osdmap --file {osdmap_file} --epoch {epoch} --force"
     cmd = cmd.format(osd=osd_path, osdmap_file=osdmap_file.name, epoch=epoch)
     ret = call(cmd, stdout=DEVNULL, shell=True)
```

Comment:
The directory rename below should only be done if any references to it are fixed as well.